data/strings/strings.txt

```diff
@@ -11356,47 +11356,49 @@
 
   [search_not_found_query]
     tags = android,ios
-    en = Try changing your search criteria.
-    af = Probeer u soekkriteria verander.
-    ar = يرجى تجربة معاير بحث أخرى.
-    az = Axtarış kriteriyalarınızı dəyişməyə cəhd edin.
-    be = Паспрабуйце змяніць крытэрыі пошуку.
-    bg = Опитайте да промените критериите за търсене.
-    ca = Proveu de canviar els criteris de cerca.
-    cs = Zkuste zadat jiný hledaný výraz.
-    da = Prøv en anden forespørgsel.
-    de = Versuchen Sie bitte eine andere Suchanfrage.
-    el = Δοκιμάστε να αλλάξετε τα κριτήρια αναζήτησης.
-    es = Por favor, pruebe con otra consulta.
-    et = Proovi muuta otsingutingimusi.
-    eu = Saiatu bilaketa-irizpideak aldatzen.
-    fa = با تغییر در معیار‌های جستجو دوباره امتحان کنید
-    fi = Yritä jotain toista hakusanaa.
-    fr = Veuillez essayer un autre mot-clé.
-    he = נסו לשנות את תנאי החיפוש.
-    hi = अपना खोज मानदण्ड बदलने का प्रयास करें।
-    hu = Próbáljon másik kifejezést.
-    id = Silakan coba kueri lainnya.
-    it = Prova con un'altra ricerca.
-    ja = 条件を変えて検索してみてください。
-    ko = 다른 쿼리를 시도하십시오.
-    lt = Pabandykite pakeisti jūsų paieškos kriterijų.
-    mr = तुमचे शोध निकष बदलण्याचा प्रयत्न करा.
-    nb = Vennligst prøv et annet søk.
-    nl = Probeer een andere zoekopdracht.
-    pl = Wpisz inne zapytanie.
-    pt = Por favor, tente outro termo de pesquisa.
-    pt-BR = Por favor, tente outro termo de busca.
-    ro = Încearcă să modifici condițiile căutării.
-    ru = Попробуйте изменить условия поиска.
-    sk = Vyskúšajte, prosím, iný hľadaný výraz.
-    sv = Försök med en annan sökförfrågan.
-    th = โปรดลองใช้คำถามอื่น
-    tr = Lütfen başka bir sorgu girin.
-    uk = Спробуйте змінити запит.
-    vi = Vui lòng thử câu hỏi khác.
-    zh-Hans = 请尝试搜索其它关键词。
-    zh-Hant = 請嘗試搜尋其它關鍵詞。
+    comment = The message when user did not find anything in the search.
+    en = Download the region where you are searching or try adding a nearby town/village name.
```

Comment:
@organicmaps/translations I've changed to more formal translations, and also rephrased to focus on downloading, and then hinting that adding a place name to the query may narrow the search result.

Reply:
`he` translation is good except that it refers to the user in male form. I try to keep it neutral most of the time and use plural as a last resort. I just realized I don't think I ever communicated this to you so I thought it's worth saying.

Gender-neutral phrasing: he = יש להוריד את האזור בו החיפוש מתבצע או להוסיף שם עיר/כפר קרוב.

Plural (I don't like this one): he = הורידו את האזור שבו אתם מחפשים או נסו להוסיף שם עיר/כפר קרוב.

I considered this quite a lot right now, and I say go with the gender-neutral option. If you have any notes about how I should deal with the user's gender, let me know.

Reply:
I assume that a gender-neutral form for HE would be the best, as we target not only male geeks ) Thanks for raising it, so we should review each HE translation that Google thinks should be male. Maybe they have some API parameter to customize it...

Reply:
Wow, all the letters have changed!
<img width="465" alt="выява" src="https://github.com/user-attachments/assets/be266547-b217-42b6-891a-6e13d50e08f0">

Reply:
A lot of documents, instructions etc. going back probably since the revival of Hebrew are written in male form, sometimes with the preface: "this document is written in male form but refers to both sexes equally". Needless to say, I don't like this approach. But it wouldn't be out of the ordinary for an app to just refer to the user in male - it's sort of the default in Hebrew. I just prefer to avoid it when I can.

I'm not quite sure what you mean by 'all the letters have changed', if there's something amusing in there I'm missing it.

Reply:
> if there's something amusing in there I'm missing it.

In other languages changing gender or using a neutral form usually is localised to a few changes (e.g. word endings).

Reply:
Right, but this isn't what I did here because then it would still be male or female. The grammar in the translation I used doesn't have an exact parallel in English, but it means something like: a download must be performed in the area where the search is done or an addition of the name of a nearby city/village. 

The verbs are discussed without mention of who's doing them, that's why it's so different. It feels more natural in Hebrew. Hope that was interesting :)